util/GenerateEndpoints.php

```diff
@@ -164,6 +359,8 @@
 
 printf("Copying the generated files to %s\n", $destDir);
 cleanFolders();
+fix_license_header($outputDir . "/Namespaces");
+fix_license_header($outputDir . "/Endpoints");
```

Comment:
Is this a 1-time thing? Meaning once the headers are fixed, will this ever run?

Generally all code should be generated from a template that already has the license header, and be 1-way, overwriting whatever we had before, rather than trying to patch/merge existing files.

Reply:
- If the endpoint exists before and have header I am keeping the same header to the newly generated corresponding endpoint. Relavant code is [here](https://github.com/opensearch-project/opensearch-php/pull/194/files#diff-bb42867ffc9abe043bb82691184bdf872c4974eddf89c134bf9faba2f2a87417R207).

    - This is necessary because the existing files contain two types of headers. For example, [security endpoints](https://github.com/opensearch-project/opensearch-php/blob/main/src/OpenSearch/Endpoints/Security/CreateTenant.php) have an OpenSearch license header, while [cat endpoints](https://github.com/opensearch-project/opensearch-php/blob/main/src/OpenSearch/Endpoints/Cat/Aliases.php) have an Elasticsearch license header.

- At last after entire generation is done, checking if any files doesnot have header and adding opensearch license header for it by calling **fix_license_header**. Corresponding code is [here](https://github.com/opensearch-project/opensearch-php/pull/194/files#diff-2ec6c40c65132791f2210be93dedd6adf95d1abcb7ea2d927f46c8a515fc0974R45). 

- Please let me know if any changes are needed.

Reply:
Makes sense.